code/headers/sound.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+namespace r2d2::sound {
+    /**
+     * Class sound a pure virtual class for sound.
+     */
+    class sound_c {
+    public:
+        /**
+         * This function starts playing a sound using a name.
+         *
+         * @param name The name of the sound you want to play.
+         */
+        virtual void start(const char *name){};
+
+        /**
+         * This function turns on the mp3 player (Also starts music)
+         *
+         */
+        virtual void power_on(){};
+
+        /**
+         * This function pauses the sound if playing.
+         *
```

Comment:
This description doesn't sound right for a power off function... Does it do the same as the pause function?